explore/src/main/scala/explore/cache/CacheModifierUpdaters.scala

```diff
@@ -41,7 +44,8 @@ trait CacheModifierUpdaters {
       )
 
   protected def modifyObservations(
-    observationEdit: ObservationEdit
+    observationEdit: ObservationEdit,
+    uuid:            UUID
```

Comment:
What's this `UUID` for?

Reply:
All of the Pots in the cache are associated with UUIDs. They are used so that the async updates only actually update if they are the most recent async request. This is to prevent updating with values that are already out of date.

Reply:
Is this a mechanism to avoid UI glitches?

If I understand correctly, the UUID is generated when a request is made and somewhere the latest UUID is stored and the result only updates the model if its UUID still matches the latest?

Can use use an existing mechanisms for this? `useSingleEffect`?